### PR TITLE
Ten cards minibridge

### PR DESCRIPTION
**Pull request by lucassperez**
Introduces the **TenCard-Minibridge** game, which is Minibridge but with a 40 card deck (ten cards for each player).

No tests added, I haven't looked at it yet, but I was able to launch some clients and successfully play a board of the game.

Let me know if you want me to eventually squash the commits! (:

### CODE REVIEW

**Comment by rulojuka** [comment]
Awesome! Thanks! :+1: 
To finalize it, just create a small unit test very similar to `Complete52CardDeckTest.java` and squash the commits to something like `[feat] add TenCardsMinibridge`

File: src/test/java/br/com/sbk/sbking/core/boarddealer/Partial40CardDeckTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    Set<Card> set = new HashSet<Card>(subject.getDeck());
+    int TOTAL_NUMBER_OF_CARDS = 40;
+    assertEquals(TOTAL_NUMBER_OF_CARDS, set.size());
+  }
+
+  @Test
+  public void getDeckshouldReturnADeckWithAllRanksGreaterThanFour() {
+    Card fourOfHearts = new Card(Suit.HEARTS, Rank.FOUR);
+    for (Card card : subject.getDeck()) {
+      boolean isGreaterThanSix = card.compareRank(fourOfHearts) > 0;
```
Comment (rulojuka) [inline src/test/java/br/com/sbk/sbking/core/boarddealer/Partial40CardDeckTest.java:36]: Shouldn't this be called `isGreaterThanFour` ?

**Review by rulojuka — CHANGES_REQUESTED** [review]

**Comment by lucassperez** [comment]
I added tests more similar to the ones in `Partial32CardDeckTest.java`, because it had an extra test that guaranteed all cards would have a rank higher than a specified value, which I thought made sense to have here, too.

The only other possible test I can think of is checking there are no duplicated cards, but I don't think it is that necessary, since the deck creation is just some for loops iterating over suits and ranks.

File: src/test/java/br/com/sbk/sbking/core/boarddealer/Partial40CardDeckTest.java (hunk 1 above)
Comment (lucassperez) [reply src/test/java/br/com/sbk/sbking/core/boarddealer/Partial40CardDeckTest.java:36]: Ops, my bad :sweat_smile:

Comment (lucassperez) [reply src/test/java/br/com/sbk/sbking/core/boarddealer/Partial40CardDeckTest.java]: Fixed. I have to be more careful with my copy and pastes :slightly_smiling_face:

Comment (rulojuka) [reply src/test/java/br/com/sbk/sbking/core/boarddealer/Partial40CardDeckTest.java]: :laughing:

**Comment by lucassperez** [comment]
About checking duplicates, the test creates a Set of cards, with solves the duplication. If the set has 40 cards, we're good.

So I think we're covered.

**Pull request merged**